proposals/gc/MVP.md

```diff
@@ -0,0 +1,314 @@
+# GC v1 Extensions
+
+See [overview](Overview.md) for background.
```

Comment:
Could you add a big bold "Note: this design is still in flux, even outside of TODOs below" or some such to the top here?

Reply:
Done.